filesystem/src/test/java/org/frankframework/filesystem/FileSystemActorCustomFileAttributesTest.java

```diff
@@ -0,0 +1,118 @@
+package org.frankframework.filesystem;
+
+import static org.frankframework.filesystem.ISupportsCustomFileAttributes.FILE_ATTRIBUTE_PARAM_PREFIX;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.core.StringContains.containsString;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import org.junit.jupiter.api.Test;
+
+import org.frankframework.parameters.IParameter;
+import org.frankframework.parameters.Parameter;
+import org.frankframework.parameters.ParameterList;
+import org.frankframework.parameters.ParameterValueList;
+import org.frankframework.stream.Message;
+
+abstract public class FileSystemActorCustomFileAttributesTest<F, S extends IWritableFileSystem<F>> extends FileSystemActorTest<F, S> {
```

Comment:
```suggestion
public abstract class FileSystemActorCustomFileAttributesTest<F, S extends IWritableFileSystem<F>> extends FileSystemActorTest<F, S> {
```
Ah wait.. here Oracle says it can be in your used order too: https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.4.3 
I just didn't know.